resource_customizations/numaflow.numaproj.io/Pipeline/actions/discovery.lua

```diff
@@ -19,28 +19,42 @@ actions["force-promote"] = {
   ["iconClass"] = "fa-solid fa-fw fa-forward"
 }
 
--- pause/unpause
-local paused = false
-if obj.spec.lifecycle ~= nil and obj.spec.lifecycle.desiredPhase ~= nil and obj.spec.lifecycle.desiredPhase == "Paused" then
-  paused = true
+function isChild(obj)
+  if obj.metadata ~= nil and obj.metadata.ownerReferences ~= nil then
+    for i, ownerRef in ipairs(obj.metadata.ownerReferences) do
+      if ownerRef["kind"] == "PipelineRollout" then
+        return true
+      end
+    end
+    return false
```

Comment:
tiny nit: you can probably remove this "return false" since it will be caught below